src/main/java/core/basesyntax/model/FruitTransaction.java

```diff
@@ -0,0 +1,68 @@
+package core.basesyntax.model;
+
+import java.util.regex.Pattern;
+
+public class FruitTransaction {
+    private Operation operation;
+    private String fruit;
+    private int quantity;
+
+    public Operation getOperation() {
+        return operation;
+    }
+
+    public FruitTransaction setOperation(Operation operation) {
+        this.operation = operation;
+        return this;
+    }
+
+    public String getFruit() {
+        return fruit;
+    }
+
+    public void setFruit(String fruit) {
+        this.fruit = fruit;
+    }
+
+    public int getQuantity() {
+        return quantity;
+    }
+
+    public void setQuantity(int quantity) {
+        this.quantity = quantity;
+    }
+
+    public static Pattern greatOperationPattern() {
```

Comment:
Consider moving the greatOperationPattern method to a separate utility class since it doesn't directly relate to the FruitTransaction properties.